src/assets/scss/modules/description.scss

```diff
@@ -1,81 +0,0 @@
-.description {
-  padding: 50px 0 100px;
-  @include backgroundImg("/assets/img/description.png");
-  text-align: center;
-
-  &__content-title{
-    margin-bottom: 14px;
-    font-size: 43px;
-    font-family: "PTSans", sans-serif;
-    color: rgb(245, 197, 6);
-    text-transform: uppercase;
-  }
-
-  &__info {
-    display: flex;
-    justify-content: flex-start;
-
-    &-free {
-      display: flex;
-      justify-content: center;
-      align-items: center;
-      position: absolute;
-      top: 0;
-      left: 0;
-      @include size(116px);
-      border-radius: 50%;
-      background-color: #f5c506;
-    }
-
-    &-images {
-      position: relative;
-      white-space: nowrap;
-      padding: 45px 0 0 20px;
-    }
-
-    &-img {
-      margin-right: 25px;
-    }
-
-    &-img:last-child {
-      margin-right: 0;
-    }
-
-    &-img:last-child {
-      vertical-align: -83px;
-    }
-
-    &-items {
-      display: flex;
-      flex-direction: column;
-      text-align: left;
-      padding: 90px 0 0 80px;
-    }
-
-    &-item {
-      position: relative;
-      margin-top: 30px;
-      padding-left: 30px;
-      font-size: 22px;
-      font-family: "OpenSans", sans-serif;
-      color: rgb(169, 169, 167);
-    }
-
-    &-item:first-child {
-      margin-top: 0;
-    }
-
-    &-item::before {
-      content: "";
-      position: absolute;
-      top: 5px;
-      left: 0;
-      display: inline-block;
```

Comment:
Why do you override default `display: block;` here? I think this rule is redundant